src/modules/logicModule/LM_PC_FundingPot_v1.sol

```diff
@@ -81,17 +108,32 @@ contract LM_PC_FundingPot_v1 is
     /// @notice The payment processor flag for the end timestamp.
     uint8 internal constant FLAG_END = 3;
 
+    /// @notice The maximum valid access criteria ID.
+    uint8 internal constant MAX_ACCESS_CRITERIA_ID = 4;
+
     // -------------------------------------------------------------------------
     // State
 
     /// @notice Stores all funding rounds by their unique ID.
     mapping(uint64 => Round) private rounds;
 
-    /// @notice Stores the access criteria ID for each round.
-    mapping(uint64 => uint8) private roundIdtoAccessId;
+    /// @notice Stores all access criteria privilages by their unique ID.
+    mapping(
+        uint64 roundId
+            => mapping(uint8 accessCriteriaId_ => AccessCriteriaPrivileges)
+    ) private accessCriteriaPrivileges;
+
+    /// @notice Maps round IDs to user addresses to contribution amounts
+    mapping(uint64 => mapping(address => uint)) private userContributions;
 
-    /// @notice The next available round ID.
-    uint64 private nextRoundId;
+    /// @notice Maps round IDs to total contributions
+    mapping(uint64 => uint) private roundTotalContributions;
+
+    /// @notice Maps round IDs to closed status
+    mapping(uint64 => bool) private roundClosed;
```

Comment:
We have a naming convention for mappings, they should include what the key is and what the value is in the rounds like 
roundIdToRound
roundItToAccessCriteriaIdToPrivileges
roundIdToUserToContribution
etc